immuneML/reports/encoding_reports/DesignMatrixExporter.py

```diff
@@ -29,18 +35,23 @@ class DesignMatrixExporter(EncodingReport):
     .. code-block:: yaml
 
         my_dme_report: DesignMatrixExporter
+            DesignMatrixExporter:
+                format_file: .npy/.csv/.hdf5/.npy.zip/.cvs.zip/.hdf5.zip
```

Comment:
here we usually had a working example, so it should be just one of these options so that users can copy-paste it if that sounds ok? :) also in other classes where we had format specified, we used them as you describe under args (without the dot). It would be nice to have it the same way in the example specs here. So maybe something like this [with an added comment to the format_file parameter and only one value]:
```
... # what you already wrote
    format_file: csv # alternatively, use one of the other formats listed above
```

Reply:
is there a way in github to add suggestions? Because I kind of see what you mean, but not sure to have understood the format precisely (I'll add a new push first soon)

Reply:
Hm, not sure, but this is what I meant how it should look [so that it can be reused as-is in this example] :)

```
my_dme_report: DesignMatrixExporter
            DesignMatrixExporter:
                format_file: csv # alternatively, use on of the formats listed above
```

Reply:
I am a bit confused now between this and the following comment

Reply:
fixed the formatting. the idea was to have an example here that could be reused (so only one option listed in the specs), while in the docstring above all options would be listed :)